Models/TrialRequestModel.cs

```diff
@@ -0,0 +1,52 @@
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace awsomAPI.Models {
+  [Table("TrialRequests")]
+  
+  public class TrialRequest {
+    [Key, Required]
+    public long Id { get; set; }
+
+    [Required]
```

Comment:
Not completely sold on having a space in between every line in here, but it's not a big deal.

Reply:
Ok, will consider this.